aldryn_django_cms/models.py

```diff
@@ -3,18 +3,29 @@
 import requests
 
 from django.conf import settings
-from django.dispatch import receiver
 
 import cms.signals
 
 
-@receiver(cms.signals.urls_need_reloading, dispatch_uid='aldryn-django-cms-cloud-apphook')
+# TODO: remove restart trigger once we've confirmed that the native restart
+#       from django-cms 3.2.x works.
+
+RESTARTER_URL = getattr(settings, 'RESTARTER_URL', None)
+RESTARTER_PAYLOAD = getattr(settings, 'RESTARTER_PAYLOAD', None)
+
+
 def trigger_server_restart(**kwargs):
     # internal endpoint to trigger safe restart
-    restarter_url = getattr(settings, 'RESTARTER_URL', None)
+    restarter_url = RESTARTER_URL
 
     if restarter_url:
```

Comment:
right... after no longer using ``getattr`` here, saving to ``restarter_url`` is pointless.